Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
+    NSCalendar* calendar = [NSCalendar currentCalendar];
+    NSDateComponents* dateComponents = [[NSDateComponents alloc] init];
```

Comment:
Nvm, ARC is enabled in this project, isn't it?

---
In reply to: [75170088](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75170088) [](ancestors = 75170088)